benches/bench.rs

```diff
@@ -54,6 +68,65 @@ fn not_refinement(c: &mut Criterion) {
     bench_non_refinement(c, "Adm2 || Researcher <= Spec // Machine");
 }
 
-criterion_group!(benches, self_refinement, refinement, not_refinement);
+fn send_query_same_components(c: &mut Criterion) {
+    c.bench_function("send_query_same_components", |b| {
+        b.to_async(FuturesExecutor).iter(|| async {
+            let backend = ConcreteEcdarBackend::default();
+            let mut responses = vec![];
+            for _ in 0..64 {
+                let request = create_query_request("determinism: Machine", 0);
+                responses.push(backend.send_query(request));
+            }
+
+            for response in responses {
+                _ = black_box(response.await);
+            }
+        });
+    });
+}
+
+fn send_query_different_components(c: &mut Criterion) {
+    c.bench_function("send_query_different_components", |b| {
```

Comment:
Plz rename b and c =)

Reply:
we dont like you

Reply:
https://bheisler.github.io/criterion.rs/book/getting_started.html
No need to thank me

Reply:
Needs to be change, like @henneboys comment

Reply:
We are not gonna do this. The other benchmarks in this same file uses this naming scheme. Either we keep it as it is now or we update all variables in this file, which is pretty out of scope of this pr.